util/dvsim/dvsim.py

```diff
@@ -109,319 +118,307 @@ def sigint_handler(signal_received, frame):
-                        default=1,
-                        metavar="N",
-                        help="""Multiplier for existing reseed values.""")
+                        metavar="PFX",
+                        help=('Prepend this string when running each tool '
+                              'command.'))
 
-    parser.add_argument("-w",
-                        "--waves",
-                        default=False,
+    parser.add_argument("--map-full-testplan",
                         action='store_true',
-                        help="Enable dumping of waves")
+                        help=("Force complete testplan annotated results to "
+                              "be shown at the end."))
 
-    parser.add_argument("-d",
-                        "--dump",
-                        default="fsdb",
-                        metavar="fsdb|shm",
-                        help="Dump waves in fsdb or shm.")
+    parser.add_argument("--publish",
+                        action='store_true',
+                        help="Publish results to reports.opentitan.org.")
 
-    parser.add_argument("-mw",
-                        "--max-waves",
+    parser.add_argument("--print-interval", "-pi",
                         type=int,
-                        default=5,
+                        default=10,
                         metavar="N",
-                        help="""Enable dumpling of waves for at most N tests;
-                                this includes tests scheduled for run AND automatic rerun"""
-                        )
+                        help="Print status every N seconds.")
 
-    parser.add_argument("-c",
-                        "--cov",
-                        default=False,
-                        action='store_true',
-                        help="turn on coverage collection")
-
-    parser.add_argument(
-        "--cov-merge-previous",
-        default=False,
-        action='store_true',
-        help="""Applicable when --cov switch is enabled. If a previous cov
-                database directory exists, this switch will cause it to be merged with
-                the current cov database.""")
-
-    parser.add_argument(
-        "--cov-analyze",
-        default=False,
-        action='store_true',
-        help="Analyze the coverage from the last regression result.")
-
-    parser.add_argument("-p",
-                        "--profile",
-                        default="none",
-                        metavar="time|mem",
-                        help="Turn on simulation profiling")
-
-    parser.add_argument("--xprop-off",
-                        default=False,
+    parser.add_argument("--max-parallel", "-mp",
+                        type=int,
+                        default=16,
+                        metavar="N",
+                        help=('Run only up to N builds/tests at a time. '
+                              'Default value 16.'))
+
+    whatg = parser.add_argument_group('Choosing what to run')
```

Comment:
This group should be moved to the top. This is pretty much what a new user would look for.